Source/DataSources/ModelGraphics.js

```diff
@@ -31,6 +31,7 @@ define([
      * @param {Property} [options.show=true] A boolean Property specifying the visibility of the model.
      * @param {Property} [options.scale=1.0] A numeric Property specifying a uniform linear scale.
      * @param {Property} [options.minimumPixelSize=0.0] A numeric Property specifying the approximate minimum pixel size of the model regardless of zoom.
+     * @param {Property} [options.maximumScale] The maximum scale size of a model. An upper limit for minimumPixelSize.
```

Comment:
Can you add this and the changes to `Model` to the 1.16 section of [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md)?